yelp_beans/routes/tasks.py

```diff
@@ -92,3 +96,31 @@ def send_match_emails():
         logging.info(matches)
         send_batch_meeting_confirmation_email(matches, spec)
     return "OK"
+
+
+@tasks.route('/init', methods=['GET'])
+def init():
```

Comment:
Are we sure this should be its own route? I feel like this is better-suited to a one-off executable that we provide.

Similar to `rake db:seed` that rails does